ChangeLog.d/psa_dh_macros.txt

```diff
@@ -0,0 +1,5 @@
+API changes
```

Comment:
All of the changes in this PR (ECC and DH) are very closely related and should be listed as a single bullet point.

> In the PSA API, rename the types of elliptic curve and Diffie-Hellman group families to psa_ecc_family_t and psa_dh_family_t, in line with the PSA Crypto API specification version 1.0.0. Rename associated macros as well.

Possibly followed by the list of macro renamings, but I didn't always bother earlier on.

Reply:
Merged both Changelist entries into one.